proxy/logging/LogFile.cc

```diff
@@ -188,9 +188,7 @@ LogFile::open_file()
       return LOG_FILE_NO_PIPE_READERS;
     }
 
-#ifdef __FreeBSD__
-    // we don't do this for FreeBSD
-#else
+#ifdef __linux__
```

Comment:
Why not F_GETPIPE_SZ?

Reply:
In either way, the doc should be fixed too.

Reply:
Should be. I was pulled by `__FreeBDS__`.